### DISCUSSION TITLE
Short code for upcoming in person and zoom meetings

### DISCUSSION THREAD

**Discussion by FresnoAAWebChair** [code4recovery/12-step-meeting-list#1149] [Q&A] [answered]
I am trying to figure out how to write a short code for in upcoming person meetings and another one for zoom meetings

This code works good ---->[tsml_next_meetings count="5"] - but I want to separated the in person and zoom meetings - I try everything I can think of but nothing is working - Any help?

**Comment by brianw-area24** [comment, marked as answer]
I just took a look at the code that generates the short codes, and I'm not seeing an option to be able to filter the short code results for online or in person meetings. If I'm missing something, someone else will chime in on this.

One of the differences between the newer TSML UI user interface, and the legacy front end is that TSML UL shows upcoming meetings at the top of the list. This might be an alternative for you, as you could link to different URLs that would filter by attendance options. This would be links to other pages, but it would filter as you've indicated.
Brian W.

**Reply by FresnoAAWebChair** [reply]
Thanxs Brian - Yes I do have filter URLs that members can click thru the top menu bar - But I was hoping to get a short code for next meetings codes to filter in-person and online - It fits nicely on my side menu widget - Thank you
